### DISCUSSION TITLE
Is PayPal support still coming?

### DISCUSSION THREAD

**Discussion by idolize** [Glench/ExtPay#87] [General]
I've had a lot of requests for this from my existing and perspective customers.

I see it's been on the roadmap/website for a while. Is this still planned to be officially supported in ExtPay?

Thanks!

**Comment by idolize** [comment]
Example https://github.com/idolize/xcloud-keyboard-mouse/issues/81

**Comment by tra4less** [comment]
i need paypal also

**Comment by amonam** [comment]
PayPal support please.
